graphics/battle_anims/backgrounds/giga_impact.pal

```diff
@@ -0,0 +1,259 @@
+JASC-PAL
+0100
+256
+115 165 115
+107 8 16
+115 8 16
+123 16 25
+132 16 25
+148 25 33
+173 16 49
+214 16 58
+255 49 107
+148 16 33
+255 148 189
+247 123 173
+239 90 140
+230 58 99
+0 0 0
```

Comment:
Color issue again: I'd say best practice is to just generate the colors from the image file itself, that works as long as it is correctly indexed.